Project.toml

```diff
@@ -16,28 +16,27 @@ AMDGPU = "21141c5a-9bdb-4563-92ae-f87d6854732e"
 CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
 FillArrays = "1a297f60-69ca-5386-bcde-b61e274b549b"
 GPUArrays = "0c68f7d7-f131-5f86-a1c3-88cf8149b2d7"
-LuxCUDA = "d0bbae9a-e099-4d5b-a835-1c6931763bda"
 Metal = "dde4c033-4e86-420c-a63e-0dd931031962"
 RecursiveArrayTools = "731186ca-8d62-57ce-b412-fbd966d074cd"
 ReverseDiff = "37e2e3b7-166d-5795-8a7a-e32c996b4267"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 Tracker = "9f7883ad-71c0-57eb-9f7f-b5c9e6d3789c"
 Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
+cuDNN = "02a925ec-e4fe-4b08-9a7e-0d78e3d38ccd"
 oneAPI = "8f75cd03-7ff8-4ecb-9b8f-daf728133b1b"
 
 [extensions]
-LuxDeviceUtilsAMDGPUExt = "AMDGPU"
-LuxDeviceUtilsCUDAExt = "CUDA"
-LuxDeviceUtilsFillArraysExt = "FillArrays"
-LuxDeviceUtilsGPUArraysExt = "GPUArrays"
-LuxDeviceUtilsLuxCUDAExt = "LuxCUDA"
-LuxDeviceUtilsMetalExt = ["GPUArrays", "Metal"]
-LuxDeviceUtilsRecursiveArrayToolsExt = "RecursiveArrayTools"
-LuxDeviceUtilsReverseDiffExt = "ReverseDiff"
-LuxDeviceUtilsSparseArraysExt = "SparseArrays"
-LuxDeviceUtilsTrackerExt = "Tracker"
-LuxDeviceUtilsZygoteExt = "Zygote"
-LuxDeviceUtilsoneAPIExt = ["GPUArrays", "oneAPI"]
+DeviceUtilsAMDGPUExt = "AMDGPU"
+DeviceUtilsCUDAExt = ["CUDA", "cuDNN"]
```

Comment:
```suggestion
DeviceUtilsCUDAExt = "CUDA"
DeviceUtilsCUDAcuDNNExt = ["CUDA", "cuDNN"]
```

Reply:
We don't list all the names in the extension only when the deps are implicit in the other package. For example, `Metal` automatically loads `GPUArrays` so we call it `MetalExt` instead of `MetalGPUArraysExt`